modules/sortie.py

```diff
@@ -396,9 +406,13 @@ def filter_retire_ship(self):
     def retire_ship(self):
         print('Retiring ship...')
         Tools.tap(self.buttons['sort'])
-        if not self.has_filtered_retire:
+        if not self.is_retire_filtered:
             self.filter_retire_ship()
-        Tools.tap(self.buttons['tobe_retired_ship'])
+        # Selecting one row botes
+        x = 130
```

Comment:
better to use `self.buttons['tobe_retired_ship']`
```suggestion
        ship = self.buttons['tobe_retired_ship']
```